web/src/components/DocumentTile.tsx

```diff
@@ -0,0 +1,69 @@
+import { Content } from "@/components/Content";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { Icon } from "@/components/njwds/Icon";
+import { ReactElement } from "react";
+
+interface Props {
+  label: string;
+  icon: string;
+  subLabel?: string;
+  downloadLink?: string;
+  isCentered?: boolean;
+  isRounded?: boolean;
+  hasExtraPadding?: boolean;
+  onClick?: () => void;
+  isWidthFull?: boolean;
+  hasDownloadIcon?: boolean;
+}
+
+export const DocumentTile = (props: Props): ReactElement => {
```

Comment:
[sand]
```typescript
export const DocumentDownloadTile = (props: Props): ReactElement => {
```

Reply:
In formation, this component is used to render the entity id. It is 1 of 4 tiles that can render on the screen. That component is not clickable, it's purely static text. Given that is the case, I think we should keep the name as is.